README.md

```diff
@@ -80,3 +82,54 @@ $ npm install
 $ npm run clean
 $ npm run build
 ```
+
+## For server-side rendering via node.js
+
+Copy `xypic.js` to your npm project, and create a `mathjax.js` in the same directory containing:
```

Comment:
You may want to indicate that it is `build/xypic.js` not `src/xypic.js`

You probably also should say that you need to do 

``` bash
npm install mathjax-full
npm install yargs
```

so that all the dependencies are in place.